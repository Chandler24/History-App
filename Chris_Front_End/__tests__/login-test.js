import 'react-native';
import React from 'react';
import Login from '../src/pages/login';
import renderer from 'react-test-renderer';

it ('renders correctly', () => {
  const snap = renderer.create(
    <Login />
  ).toJSON();
  expect(snap).toMatchSnapshot();
});