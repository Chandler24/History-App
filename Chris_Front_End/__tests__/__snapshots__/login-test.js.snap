// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2e88ff",
      "flexGrow": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "margin": 50,
      }
    }
  />
  <View>
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Username"
      placeholderTextColor="rgba(255,255,255,0.75)"
      selectionColor="rgba(255,255,255,0.75)"
      style={
        Object {
          "backgroundColor": "rgba(255,255,255,0.5)",
          "borderRadius": 25,
          "color": "#ffffff",
          "height": 40,
          "marginBottom": 20,
          "paddingHorizontal": 20,
          "width": 300,
        }
      }
      underlineColorAndroid="rgba(0,0,0,0)"
      value=""
    />
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Password"
      placeholderTextColor="rgba(255,255,255,0.75)"
      secureTextEntry={true}
      selectionColor="rgba(255,255,255,0.75)"
      style={
        Object {
          "backgroundColor": "rgba(255,255,255,0.5)",
          "borderRadius": 25,
          "color": "#ffffff",
          "height": 40,
          "marginBottom": 20,
          "paddingHorizontal": 20,
          "width": 300,
        }
      }
      underlineColorAndroid="rgba(0,0,0,0)"
      value=""
    />
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#005ccb",
          "borderRadius": 25,
          "height": 50,
          "opacity": 1,
          "paddingVertical": 9,
          "width": 300,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#ffffff",
            "fontSize": 20,
            "fontWeight": "500",
            "textAlign": "center",
          }
        }
      >
        Log In
      </Text>
    </View>
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      onPress={[Function]}
      style={
        Object {
          "alignItems": "flex-end",
          "color": "#ffffff",
          "justifyContent": "center",
          "marginTop": 20,
        }
      }
    >
      Don't have an Account? Sign up
    </Text>
  </View>
</View>
`;
